source/content/optimize-site-traffic.md

```diff
@@ -167,12 +167,12 @@ Install and use one of the following WordPress plugins:
 
 Similar to the IP blocking methods listed above, you can also target specific unwanted user agents that you may want to block.
 
-The `stripos` function implements a case-insenstive match which can be helpful when dealing with mixed bots or crawlers, such as `Curl/dev` vs `curlBot`. 
+The `stripos` function implements a case-insenstive match which can be helpful when dealing with mixed bots or crawlers, such as `Curl/dev` vs `curlBot`.
 
-Remember to replace the example user agent (`UglyBot`):
+Remember to replace the example user agents (`UglyBot` and `PetalBot`):
 
 ```php:title=wp-config.php%20or%20settings.php
-if (stripos($_SERVER['HTTP_USER_AGENT'], 'UglyBot') !== FALSE) {
+if (stripos($_SERVER['HTTP_USER_AGENT'], 'UglyBot' || 'PetalBot') !== FALSE) {
```

Comment:
We should probably just make a loop here if we want an example that shows blocking multiple user agents by substring. There's one-liner ways to do it but they're fairly obtuse and I'd vote for verbosity and clarity in an example like this.

Was this form tested?

Reply:
I updated it with an `else`, but I haven't found a way to test it on my own [L180](https://github.com/pantheon-systems/documentation/pull/5739/files#diff-a43b4f6f8cd58b0d87a62563f4a1e2e5R180)